pkg/controller/useraccount/useraccount_controller.go

```diff
@@ -107,16 +110,37 @@ func (r *ReconcileUserAccount) Reconcile(request reconcile.Request) (reconcile.R
 		return reconcile.Result{}, err
 	}
 
-	var createdOrUpdated bool
-	var user *userv1.User
-	if user, createdOrUpdated, err = r.ensureUser(reqLogger, userAcc); err != nil || createdOrUpdated {
-		return reconcile.Result{}, err
-	}
+	// Check if the UserAccount has been deleted. If it has been deleted, delete
```

Comment:
this comment is for `else` part and not for `if` part.  I guess, it's due to refactoring code and swap if-else but you forgot to move comment :-(

Reply:
Sorry, i didn't forget to move the comment. I left it there to explain the else block and there was no comment for the addition of user/identity. But i can add one to the comment